presto-orc/src/main/java/com/facebook/presto/orc/OrcSelectiveRecordReader.java

```diff
@@ -160,6 +164,20 @@ public OrcSelectiveRecordReader(
 
         requireNonNull(constantValues, "constantValues is null");
         this.constantValues = new Object[this.hiveColumnIndices.length];
+        for (Integer columnIndex : includedColumns.keySet()) {
+            if (!presentColumns.contains(columnIndex)) {
```

Comment:
use isColumnPresent method from the base class: if (!isColumnPresent(columnIndex)) {